Functions/Functions/History/GenerateHistoryFileFunction.cs

```diff
@@ -35,10 +37,10 @@ public GenerateHistoryFileFunction(RulesDbContext dbContext)
                 ruleHistory.Add(new RuleHistoryData
                 {
                     file = history.MarkdownFilePath,
-                    lastUpdated = history.ChangedAtDateTime,
+                    lastUpdated = history.ChangedAtDateTime.ToString(dateFormat),
```

Comment:
Use CultureInfo in both:

`history.ChangedAtDateTime.ToString(dateFormat, CultureInfo.InvariantCulture)`

Reply:
**Done** - Added CultureInfo on DateTime to String